packages/hub/src/app/models/[owner]/[slug]/EditSquiggleSnippetModel.tsx

```diff
@@ -111,8 +89,42 @@ export const EditSquiggleSnippetModel: FC<Props> = ({ modelRef }) => {
     };
   }, [content, revision.relativeValuesExports]);
 
-  const form = useForm<FormShape>({
+  const { form, onSubmit, inFlight } = useMutationForm<
+    FormShape,
+    EditSquiggleSnippetModelMutation,
+    "UpdateSquiggleSnippetResult"
+  >({
     defaultValues: initialFormValues,
+    mutation: graphql`
```

Comment:
I'm still not sure if inlining mutations is good or bad, but started to lean towards it being good. Mutation name (`EditSquiggleSnippetModelMutation`) and its expectedTypename (`UpdateSquiggleSnippetResult`) need to be mentioned in generics and other params.

So if the mutation was declared separately outside of the component, you'd have to jump back and forth to change it.